IndomitableIbises/Bangazon-Sprint-2
# Review ComputerDetailView and ComputerFormView

## Link to Ticket
References # [Link](https://github.com/IndomitableIbises/Bangazon-Sprint-2/issues/8)

## Description of Proposed Changes

- This ticket creates a  computer detail and form view

## Steps to Test

Outline the steps to test

```sh
1. git fetch --all
2. git checkout computerModel-ea
3. rm -rf migrations folder
4. rm -rf db.sqlite3
5. python manage.py makemigrations bang
6. python manage.py migrate
7. python manage.py runserver
8. from main website view choose "computers"
9. choose "add new" and save
10.  You should be able to view a list of all computers after save
11.  You should be able to click hyperlink on a computer to view details
```

## Impacted Areas in Application

List general components of the application that this PR will affect:

*Idk

## Mentions @username

anyone who wants to view

## Definition of Done

1. The project must be fully documented. This includes the following:
   1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
   1. Every class must be documented with purpose, author, and methods.
   1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each
